Front page rework and consistency update (#170)
SlimeVR/SlimeVR-Docs-Site: src/README.md

## What is SlimeVR?

SlimeVR is a set of open hardware sensors and open source software that facilitates full-body tracking (FBT) in virtual reality. The project is built around creating a system that is customisable, hackable and can be modified to suit the needs of the user. Due to this, this documentation provides the full breakdown of not just how to set up your environment, but also how you could build a set of trackers yourself. For a more detailed breakdown of how the system works, check [SlimeVR 101](slimevr101.html).

## How do I get my hands on those sweet sweet SlimeVR trackers?

There are currently multiple ways to approach putting together your own FBT solution using SlimeVR.

### 1. Buying trackers

#### Fully built trackers directly from SlimeVR

![Slime Trackers](assets/img/slimeVRTrackers.jpg)

You buy fully built trackers for pre-order at [Crowd Supply](https://www.crowdsupply.com/slimevr/slimevr-full-body-tracker). These trackers are the passion project of dedicated core members of SlimeVR, and due to chip shortages, shipping delays and the like we cannot guarantee shipping dates or turnaround time.

> This option is currently estimated to begin shipping in July of 2023 (the earliest orders shipped first) with limited supply.

Reviewer: ```suggestion
This option is a preorder. Please check the product page for extimated shipping times for new orders. Actual shipping times might differ due to production delays and other circumstances.
```

#### 3rd Party sellers

> 3rd Party sellers are becoming common, with commissions being offered on the [SlimeVR discord](https://discord.gg/SlimeVR) marketplace forum. Designs and specifications vary from offering to offering, so make sure you to check exactly what you are getting! As one of the most important aspects of the tracker is the IMU (used to measure your movement), we suggest you check the [IMU comparison page](diy/imu-comparison.html) to get an idea of what to expect from any available tracker.

Reviewer: ```suggestion
3rd Party sellers are becoming common, with commissions being offered on the [SlimeVR discord](https://discord.gg/SlimeVR) marketplace forum. Designs and specifications vary from offering to offering, so make sure to check exactly what you are getting! As one of the most important aspects of the tracker is the IMU (used to measure your movement), we suggest you check the [IMU comparison page](diy/imu-comparison.html) to get an idea of what to expect from any available tracker.
```

> > **Caution:** SlimeVR cannot ensure 3rd party trackers meet any requirements of quality, please assume that purchasing from a 3rd party seller is the equivalent of buying from a small creator and your own research into the quality of these trackers is important. We suggest you check reviews or talk to others that have purchased from the seller in order to get an idea of what to expect. If any of your purchased 3rd party trackers develop a fault, contact the seller for help. However, you may need some understanding of soldering in order to repair them yourself.

Reviewer: ```suggestion
> **Caution:** SlimeVR cannot ensure 3rd party trackers meet any requirements of quality, please assume that purchasing from a 3rd party seller is the equivalent of buying from a small creator, and your own research into the quality of these trackers is important. We suggest you check reviews or talk to others that have purchased from the seller in order to get an idea of what to expect. If any of your purchased 3rd party trackers develop a fault, contact the seller for help. However, you may need some understanding of soldering in order to repair them yourself.
```

### 2. Building them yourself

#### Entirely from scratch

![Example DIY build](assets/img/exampleBuild.jpg)<br>
*Example build by NightyIceC00kie*

Building trackers is the most common way users have trackers at the time of writing. This documentation provides a full [guide to building trackers from scratch](diy/index.html), which provides lists of full components needed, where to buy them and schematics for many combinations of IMUs and Microcontrollers.

This is the cheapest and fastest way to obtain trackers, assuming you have a soldering iron on hand a single tracker can be put together for under $20. 

DIY builds require time to assemble and may require self-repairs from time to time.

#### Pre-order the official DIY Kit on Crowd Supply

![DIY kit](assets/img/diyKit.jpg)<br>
*A prototype of DIY-Kit boards and wires.*

> You buy the [**official DIY Kit**](https://www.crowdsupply.com/slimevr/slimevr-full-body-tracker) which includes all the boards and cables you need, get a LiPo battery, and attach batteries to the boards. All you need is to 3D print (or build) yourself a nice casing and get straps (or get creative and tape them with electrical tape to your body).

Reviewer: ```suggestion
You buy the [**official DIY Kit**](https://www.crowdsupply.com/slimevr/slimevr-full-body-tracker) which includes all the boards and cables you need, get a LiPo battery, and attach batteries to the boards. All you need is to 3D print (or build) yourself a nice casing and get straps (or get creative and tape them with electrical tape to your body). See [DIY Kit Guide](/diy_kit_guide.html) for more information on how to build it.
```

> This option requires no or little soldering, provides tested boards, allows for the smallest possible design, and is a cheaper alternative to buying completed SlimeVR trackers. It also allows you to customize cases and straps.

Reviewer: ```suggestion
This option requires no or little soldering, provides tested boards, best available IMUs, allows for a small design, and is a cheaper alternative to buying completed SlimeVR trackers. It also allows you to customize cases and straps.
```

However, at this point you could just buy completed SlimeVR trackers as the shipping times and chip shortages still affect this option. 

> This option is currently estimated to begin shipping in July of 2023 (the earliest orders shipped first) with limited supply.

Reviewer: ```suggestion
This option is a preorder. Please check the product page for extimated shipping times for new orders. Actual shipping times might differ due to production delays and other circumstances.
```
